thirdparty.spdx

```diff
@@ -37,8 +37,8 @@ SPDXID: SPDXRef-jsoncpp
 PackageVersion: 0.6.0
 PackageDownloadLocation: https://github.com/open-source-parsers/jsoncpp/archive/1.6.0.zip
 PackageHomePage: https://github.com/open-source-parsers/jsoncpp
-PackageLicenseConcluded: LicenseRef-jsoncpp
-PackageLicenseDeclared: LicenseRef-jsoncpp
```

Comment:
jsoncpp license has following prefix:

The JsonCpp library's source code, including accompanying documentation,
tests and demonstration applications, are licensed under the following
conditions...

The author (Baptiste Lepilleur) explicitly disclaims copyright in all
jurisdictions which recognize such a disclaimer. In such jurisdictions,
this software is released into the Public Domain.

In jurisdictions which do not recognize Public Domain property (e.g. Germany as of
2010), this software is Copyright (c) 2007-2010 by Baptiste Lepilleur, and is
released under the terms of the MIT License (see below).

In jurisdictions which recognize Public Domain property, the user of this
software may choose to accept it either as 1) Public Domain, 2) under the
conditions of the MIT License (see below), or 3) under the terms of dual
Public Domain/MIT License conditions described here, as they choose.